docker-compose.yml

```diff
@@ -2,13 +2,17 @@ version: '2.1'
 
 services:
   fact-api:
```

Comment:
I don't normally put the service itself in here, I use docker-compose for the dependencies and then run the app via gradle/intellij. Don't mind if you want to include it though

Reply:
It was in the template app, I think leaving it in gives another option